<ui version="4.0" >
 <class>GraphWidgetCtrlClass</class>
 <widget class="QWidget" name="GraphWidgetCtrlClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>4000</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>GraphWidgetCtrl</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="margin" >
    <number>2</number>
   </property>
   <item>
    <widget class="QPushButton" name="btnMode" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>Selection Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnAssignCluster" >
     <property name="text" >
      <string>Assign To...</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnShowEdge" >
     <property name="text" >
      <string>Show Edges</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDispMode" >
     <property name="text" >
      <string>Display Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnZoomIn" >
     <property name="text" >
      <string>Zoom In</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnFit" >
     <property name="text" >
      <string>Fit Window</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
